bentoml/yatai/deployment/aws_lambda/lambda_app.py

```diff
@@ -45,11 +49,11 @@
 if not os.path.exists(bento_bundle_path):
     bento_bundle_path = os.path.join('/tmp/requirements', bento_name)
 
-print(f'Loading BentoService bundle from path: "{bento_bundle_path}"')
+logger.info('Loading BentoService bundle from path: "%s"', bento_bundle_path)
```

Comment:
how about changing all these to `logger.debug` and only keep the prediction log in `logger.info`?